README.md

```diff
@@ -1,4 +1,4 @@
-# Adafruit FONA Library[![Build Status](https://github.com/adafruit/Adafruit_FONA/workflows/Arduino%20Library%20CI/badge.svg)](https://github.com/adafruit/Adafruit_FONA/actions)
+# Adafruit FONA Library[![Build Status](https://github.com/adafruit/Adafruit_FONA/workflows/Arduino%20Library%20CI/badge.svg)](https://github.com/adafruit/Adafruit_FONA/actions)[![Documentation](https://github.com/adafruit/ci-arduino/assets/doxygen_badge.svg)](http://adafruit.github.io/Adafruit_FONA/html/index.html)
```

Comment:
this is the proper path; you have to include the branch name:
https://github.com/adafruit/ci-arduino/blob/master/assets/doxygen_badge.svg